tests/utils/runner/network_manager.go

```diff
@@ -66,12 +66,14 @@ func NewDefaultANRConfig() ANRConfig {
 		AvalancheGoExecPath: os.ExpandEnv("$GOPATH/src/github.com/ava-labs/avalanchego/build/avalanchego"),
 		PluginDir:           os.ExpandEnv("$GOPATH/src/github.com/ava-labs/avalanchego/build/plugins"),
 		GlobalNodeConfig: `{
+			"log-level":"info",
```

Comment:
Should we remove `log-display-level` if we are going to specify `log-level`? Also, not sure why we were originally specifying `log-display-level` here (although it looks like I wrote it, I think I copied it from somewhere).

Most likely this was to log to stdout without writing to the log directory. I'd assume if anything we'd want more specific logs written to stdout, so that we can inspect them on test failure without crowding out the output from the network runner, which pipes the logs from stdout (as I understand it).

Reply:
removed